modules/social_features/social_like/social_like.tokens.inc

```diff
@@ -77,12 +77,13 @@ function social_like_tokens($type, $tokens, array $data, array $options, Bubblea
 
             $content_type = $entity->getEntityTypeId();
             // Check if the content type is node.
+            $content_type_label = '';
             if ($content_type === 'node') {
               // Then get the bundle name.
-              $content_type_label = mb_strtolower(\Drupal::entityTypeManager()
-                ->getStorage('node_type')
-                ->load($entity->bundle())
-                ->label());
+              $label = \Drupal::entityTypeManager()->getStorage('node_type')->load($entity->bundle())->label();
+              if (is_string($label) || $label instanceof TranslatableMarkup) {
```

Comment:
Interesting, if $label instanceof TranslatableMarkup, shouldn't it then be the ->render() instead?